package.json

```diff
@@ -16,20 +16,22 @@
     "release-to-branch": "node bin/check-nvmrc.js && ./bin/release-to-branch.sh",
     "build:package": "node bin/check-nvmrc.js && gulp build:package --destination 'package' && npm run test:build:package",
     "build:dist": "node bin/check-nvmrc.js && gulp build:dist --destination 'dist' && npm run test:build:dist",
-    "test": "standard && gulp test && gulp copy-assets && jest --testPathIgnorePatterns='after-*'",
+    "test": "npm ls --depth=0 && standard && gulp test && gulp copy-assets && jest",
```

Comment:
Could we either add a comment or break this into its own named task to make it clear what this is doing? I initially thought that this had been left in from debugging, as I wouldn't expect the exit code of `npm ls` to be important.

Reply:
Yes will do.

It checks the dependency tree for missing packages (you had a handful).

Can't put comments in JSON so I'll make the script name more descriptive.

Reply:
@36degrees I've resolved this with a new `npm run test:dependencies` task that's run first before any tests are started.